nucypher/types.py

```diff
@@ -92,3 +92,18 @@ class StakerInfo(NamedTuple):
     # downtime: Tuple[Downtime, ...]
     # substake_info: Tuple[RawSubStakeInfo, ...]
     # history: Tuple[NuNits, ...]
+
+
+class Policy(NamedTuple):
+    disabled: bool
+    sponsor: ChecksumAddress
+    owner: ChecksumAddress
+    fee_rate: Wei
+    start_timestamp: int
+    end_timestamp: int
+
+    # reserved_slot_1
```

Comment:
In what sense is it reserved?

Reply:
Ah, I see. Could you add a short comment here, something along the lines of "reserved but unused fields in the corresponding Solidity structure"?

Reply:
✔️